src/Surfnet/StepupMiddleware/CommandHandlingBundle/EventListener/CommandProcessingExceptionListener.php

```diff
@@ -18,13 +18,13 @@
 
 namespace Surfnet\StepupMiddleware\CommandHandlingBundle\EventListener;
 
-use Surfnet\StepupMiddleware\CommandHandlingBundle\Pipeline\Exception\InvalidCommandException;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\Pipeline\Exception\ProcessingAbortedException;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
 use Symfony\Component\HttpKernel\KernelEvents;
 
-class InvalidCommandExceptionListener implements EventSubscriberInterface
+class CommandProcessingExceptionListener implements EventSubscriberInterface
```

Comment:
Will this handle all CommandProcessingExceptions? I.e. when a DomainException is thrown? Which exceptions can I expect it to handle?